api/base/serializers.py

```diff
@@ -135,6 +137,90 @@ def to_representation(self, value):
         return {'links': {self.link_type: {'href': url, 'meta': meta}}}
 
 
+class JSONAPIHyperlinkedRelatedField(ser.HyperlinkedRelatedField):
```

Comment:
As we add more custom field types, would it make sense to split them into a separate file (`api/base/fields.py`) to keep the code organized?